src/execution/execution.types.ts

```
@@ -0,0 +1,15 @@
+export interface ExecutionConfig {
+  outputFormat: string;
+  threshold: number;
+  useSemanticScoring: boolean;
+  // Define the properties needed for execution configuration
+  taskType: string;
+  parameters: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)